package.json

```diff
@@ -1,18 +1,21 @@
 {
   "name": "@burdaforward/composable-rules",
-  "version": "3.0.0",
+  "version": "3.0.1",
   "description": "A library for composable rule logic. Easily combine simple rules into more complex ones in a maintainable way using your custom logic and data.",
-  "unpkg": "./dist/umd/index.js",
-  "main": "./dist/cjs/index.js",
-  "types": "./dist/types/index.d.ts",
-  "module": "./dist/esm/index.js",
+  "main": "./dist/index.cjs",
+  "unpkg": "./dist/index.js",
+  "module": "./dist/index.js",
+  "types": "./dist/index.d.ts",
   "exports": {
-    "require": "./dist/cjs/index.js",
-    "default": "./dist/esm/index.js"
+    ".": {
+      "require": "./dist/index.cjs",
+      "import": "./dist/index.js",
+      "default": "./dist/index.js",
+      "types": "./dist/index.d.ts"
```

Comment:
I read once that exports are checked in order. `default` always matches so it needs to go last. Otherwise `types` will never be matched.